index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
                 </a>
                 <ul class="icons">
                     <li><a href="https://twitter.com/carlosigreda" target="_blank" rel="noopener"><img
-                                src="/images/Twiter.svg" alt="Twiter"></a></li>
+                                src="./images/Twiter.svg" alt="Twiter"></a></li>
                     <li><a href="https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/" target="_blank"
-                            rel="noopener"><img src="/images/Linkedin.svg" alt="Linkedin"></a></li>
-                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="/images/Medium.svg"
+                            rel="noopener"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="./images/Medium.svg"
                                 alt="Medium"></a>
                     </li>
                     <li><a href="https://github.com/CarlosIgreda" target="_blank" rel="noopener"><img
-                                src="/images/Github.svg" alt="Github"></a></li>
+                                src="./images/Github.svg" alt="Github"></a></li>
                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener"><img
-                                src="/images/AngelList.svg" alt="AngelList"></a></li>
+                                src="./images/AngelList.svg" alt="AngelList"></a></li>
                 </ul>
             </div>
         </section>
+        <section class="work s1">
+            <img src="./images/s1.svg" alt="tonic">
+            <h1>Tonic</h1>
```

Comment:
- [ ] kindly use an h2 tag here since its a sub heading  .. Using only one H1 creates hierarchy and gives your page a more logical structure. It makes it easy for readers  to skim and understand what your page is about. please apply to all sections

Reply:
Done @tufoinnkuo10 . I've already changed it to an h2 tag.